core-it-suite/src/test/java/org/apache/maven/it/MavenITmng0768OfflineModeTest.java

```diff
@@ -103,17 +103,12 @@ else if ( request.getRequestURI().endsWith( ".md5" ) || request.getRequestURI().
         try
         {
             server.start();
-            while ( !server.isRunning() || !server.isStarted() )
+            if ( server.isFailed() )
```

Comment:
This state is not possible to reach immediately after `server.start()` in a Bind failure scenario.

The ServerConnector's doStart will attempt to bind
The BindException gets thrown for failure to bind
A IOException wraps failure to bind during ServerConnector doStart
The Server doStart captures the IOException and adds it to the MultiException its tracking
At the end of Server doStart the MultiException throws if it has any exceptions its tracking.

This scenario should result in a IOException or MultiException from server.start()